subprojects/gst-plugins-bad/sys/msdk/gstmsdkdec.c

```diff
@@ -1889,6 +1945,12 @@ gst_msdkdec_class_init (GstMsdkDecClass * klass)
           1, 20, PROP_ASYNC_DEPTH_DEFAULT,
           G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
 
+  g_object_class_install_property (gobject_class, GST_MSDKDEC_PROP_DEC_POSTPROC,
+      g_param_spec_boolean ("dec-postproc", "Dec Postproc",
```

Comment:
Not sure I like this prop name (i.e. `dec` is already implied because it is a decoder element).  It only enables postproc csc, right? How about `vpp-csc`?  Other suggestions?

Reply:
This prop name is to indicate that this processing is applied in decoder through a buffer called mfxExtDecVideoProcessing, which is different from vpp implementation in mediaSDK/oneVPL . Also, sample_decode use "dec_postproc" as a prop name to enable this sfc feature, so I use this name as well to align with mediaSDK/oneVPL.  vpp-csc may not fit very well because this prop is not only for csc, it can be used for resizing as well (currently not implemented scaling here).

Reply:
How about `sfc`.  Not all props in gstreamer have to have 1:1 name mapping to mfx or sample_decode.

Reply:
IDK, we can also see what upstream reviewers say.

Reply:
It might be ok.

Reply:
sfc could be a good option. Let's see what is the opinion from the community, may also discuss name there. 
Thanks for your suggestion :)